Reduce amount of syslog traffic
- reduce "coverage" log message from WARN->INFO
- Only log WARN and higher to syslog
- Log INFO and higher to /var/log/ovs-{vswitchd,brcompatd}.log

diff -r 19c86a17fb07 lib/timeval.c
--- a/lib/timeval.c	Fri Sep 11 10:53:51 2009 +0100
+++ b/lib/timeval.c	Fri Sep 11 14:37:10 2009 +0100
@@ -294,7 +294,7 @@
                       rusage.ru_nvcsw - last_rusage->ru_nvcsw,
                       rusage.ru_nivcsw - last_rusage->ru_nivcsw);
         }
-        coverage_log(VLL_WARN, true);
+        coverage_log(VLL_INFO, true);
     }
 
     /* Update exponentially weighted moving average.  With these parameters, a
diff -r 19c86a17fb07 xenserver/etc_init.d_vswitch
--- a/xenserver/etc_init.d_vswitch	Fri Sep 11 10:53:51 2009 +0100
+++ b/xenserver/etc_init.d_vswitch	Fri Sep 11 14:37:10 2009 +0100
@@ -35,7 +35,7 @@
 : ${VSWITCHD_RUN_DIR:=/var/xen/vswitch}
 : ${VSWITCHD_PRIORITY:=-10}
 : ${VSWITCHD_LOGFILE:=/var/log/ovs-vswitchd.log}
-: ${VSWITCHD_FILE_LOGLEVEL:=}
+: ${VSWITCHD_FILE_LOGLEVEL:=INFO}
 : ${VSWITCHD_SYSLOG_LOGLEVEL:=WARN}
 : ${VSWITCHD_MEMLEAK_LOGFILE:=}
 : ${VSWITCHD_STRACE_LOG:=}
@@ -48,8 +48,8 @@
 : ${BRCOMPATD_RUN_DIR:=/var/xen/vswitch}
 : ${BRCOMPATD_PRIORITY:=-10}
 : ${BRCOMPATD_LOGFILE:=/var/log/ovs-brcompatd.log}
-: ${BRCOMPATD_FILE_LOGLEVEL:=}
-: ${BRCOMPATD_SYSLOG_LOGLEVEL:=INFO}
+: ${BRCOMPATD_FILE_LOGLEVEL:=INFO}
+: ${BRCOMPATD_SYSLOG_LOGLEVEL:=WARN}
 : ${BRCOMPATD_MEMLEAK_LOGFILE:=}
 : ${BRCOMPATD_STRACE_LOG:=}
 : ${BRCOMPATD_STRACE_OPT:=}
@@ -166,7 +166,7 @@
         mkdir -p "$BRCOMPATD_RUN_DIR"
     fi
     cd "$BRCOMPATD_RUN_DIR"
-    if [ ! -n "$BRCOMPATD_FILE_LOGLEVEL" ]; then
+    if [ -n "$BRCOMPATD_FILE_LOGLEVEL" ]; then
         logfile_level_opt="-vANY:FILE:${BRCOMPATD_FILE_LOGLEVEL}"
         logfile_file_opt="--log-file=$BRCOMPATD_LOGFILE"
     fi
diff -r 19c86a17fb07 xenserver/etc_logrotate.d_vswitch
--- a/xenserver/etc_logrotate.d_vswitch	Fri Sep 11 10:53:51 2009 +0100
+++ b/xenserver/etc_logrotate.d_vswitch	Fri Sep 11 14:37:10 2009 +0100
@@ -5,7 +5,7 @@
 # notice and this notice are preserved.  This file is offered as-is,
 # without warranty of any kind.
 
-/var/log/ovs-vswitchd.log {
+/var/log/ovs-vswitchd.log /var/log/ovs-brcompatd.log {
 	sharedscripts
 	postrotate
         # Send sighup to vswitch which will cause it to reopen its log files.
